kdc/kx509.c

```diff
@@ -460,7 +412,7 @@ mk_error_response(krb5_context context,
         msg = freeme1;
     }
 
-    kdc_log(context, reqctx->config, 1, "%s", msg);
+    /* XXX Call _kdc_audit_addreason() here to save our callers the trouble */
```

Comment:
Is that still pertinent?

Reply:
Just a code improvement suggestion, but no.